package.json

```diff
@@ -8,12 +8,18 @@
     "!/contracts/mocks/**/*",
     "/dist/**/*"
   ],
-  "main": "dist/src/index.js",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.esm.js",
+  "types": "dist/src/index.d.ts",
+  "exports": {
+    "require": "./dist/index.cjs.js",
+    "import": "./dist/index.esm.js"
+  },
```

Comment:
that works for both?

Reply:
Yep uses conditional exports so it can be consumed as either esm or commonjs: https://nodejs.org/api/packages.html#conditional-exports